rasa/shared/core/domain.py

```diff
@@ -185,7 +184,6 @@ def from_dict(cls, data: Dict) -> "Domain":
     def _get_session_config(session_config: Dict) -> SessionConfig:
         session_expiration_time_min = session_config.get(SESSION_EXPIRATION_TIME_KEY)
 
-        # TODO: 2.0 reconsider how to apply sessions to old projects and legacy trackers
```

Comment:
I guess we've just decided to leave them as is?

Reply:
No, I changed the default from `0` to `60` which essentially enables them by default.

Reply:
i guess i assumed that this note was about "should we read all of the conversations from 2018 as one conversation even if we introduce sessions now? with regards to applying them to legacy trackers

Reply:
Ah, okay. Yes, these will have sessions enabled, too, now